### PR TITLE
KP-8873 configure backup of /www

### PR DESCRIPTION
**Pull request by mmatthiesencsc**
added daily backup of /www

### CODE REVIEW

File: servers/sanat/site.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@
 - hosts: web
   become: yes
   roles:
+    - role: kielipankki.common.backup
```
Comment (Nikerabbit) [inline servers/sanat/site.yml:10]: Should this be listed as dependency?

Comment (mmatthiesencsc) [reply servers/sanat/site.yml:10]: It is part of this: https://github.com/CSCfi/Kielipankki-palvelut/blob/cc3161067239c95d041a87d3c521f39e973f78d7/servers/sanat/requirements-pouta_creation.yml#L4C11-L4C62

Comment (Nikerabbit) [reply servers/sanat/site.yml:10]: Okay.

**Review by aajarven — COMMENTED** [review]
Looks good. If we do something about https://github.com/CSCfi/Kielipankki-ansible-common/pull/26/files#r1810461843, then the `backup_db_password` variable might change to something more descriptive for this use though.

**Pull request merged**